Dockerfile.dev

```diff
@@ -1,3 +1,4 @@
+{'from': [{'source': 'openshift-golang-builder\\:v(?:[\\.0-9\\-]*).*', 'target': 'registry.ci.openshift.org/openshift/release:golang-1.14 AS builder'}, {'source': 'openshift-ose-base\\:v(?:[\\.0-9\\-]*)', 'target': 'docker.io/centos:8 AS centos'}], 'env': [{'source': 'RUNBOOK_BASE_URL=.*', 'target': 'RUNBOOK_BASE_URL="https://github.com/openshift/elasticsearch-operator/blob/master/docs/alerts.md"'}]}
```

Comment:
Is this block intended or a generation mistake?

Reply:
good catch, i think this is a generation issue and why the images aren't building

Reply:
i left a debugging print statement in the generate script was the issue...